### DISCUSSION TITLE
Name.com doesn't accept the last digit in custom.plausible.io.

### DISCUSSION THREAD

**Discussion by ricotrevisan** [plausible/analytics#742] [Anything Else]
I've got my site set up on Webflow with the domain from Name.com. When I try to establish the custom domain it won't let me set it up as instructed on Plausible. Any ideas? It only works without the `.` at the end.

![image](https://user-images.githubusercontent.com/2459327/108680105-bf456780-74ed-11eb-9f98-8b09b1a4ae3b.png)

One idea I'm throwing around is to simply move the domain name to another service (this is not the first issue I have with name.com). Cloudflare? Hover? Is there one that the hive mind recommends?

**Comment by metmarkosaric** [comment]
Thanks @RicoTrevisan! It should work without the last dot as well. Try it like that please.

All the instructions are here: https://plausible.io/docs/custom-domain
